lib/wakes/location_helper.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+class LocationHelper
```

Comment:
@chrismaddocks a question and a few thoughts here:

1.) Is this only used in the matcher?

2.) I'm generally not a fan of the name. 

3.) It looks like you are substantially re-implementing the `URI()` method. You might be able to get away with a clever use of `URI()` instead of this class, or you might look into making this class a thin wrapper around `URI()`